R/tidyverse.R

```diff
@@ -137,10 +137,9 @@ dep_version <- function(x) {
 #' @export
 #' @rdname tidyverse
 use_tidy_eval <- function() {
-  if (!uses_roxygen()) {
-    stop_glue("{code('use_tidy_eval()')} requires that you use roxygen.")
-  }
+  check_is_package("use_tidy_eval()")
 
+  use_dependency("roxygen2", "Suggests")
```

Comment:
I don't see why we required evidence of prior use of roxygen2. Seems like we can just make sure it is, at least, in Suggests and carry on.

Conversely, I think we should make sure we're dealing with a package.